lib/src/checkbox/Checkbox.tsx

```diff
@@ -67,47 +76,46 @@ const DxcCheckbox = ({
         </>
       ) : (
         <>
-          {optional && <span>(Optional)</span>} {label}
+          {optional && <span>{translatedLabels.formFields.optionalLabel}</span>} {label}
         </>
       )}
     </LabelContainer>
   );
 
   return (
     <ThemeProvider theme={colorsTheme.checkbox}>
-      <CheckboxContainer
-        id={name}
-        brightness={componentTokens}
-        label={label}
-        labelPosition={labelPosition}
-        disabled={disabled}
-        margin={margin}
-        size={size}
-        backgroundType={backgroundType}
-        isLabelHovered={isLabelHovered}
-      >
+      <CheckboxContainer id={name} disabled={disabled} margin={margin} size={size}>
         {label && labelPosition === "before" && labelComponent}
-        <Checkbox
+        <CheckboxInput
```

Comment:
Some comments about this checkbox input. I think we should approach this refactor as the Select, let me explain. The custom checkbox is CheckboxFocus because we do not want to use the native one. So that is what the user sees and it should have the `role="checkbox" name={name} aria-labelledby={labelId} aria-checked={checked ?? innerChecked}`. The input should be only used for the submit event and must be hidden from the user, both visually and in the DOM. 

```
<ValueInput
  name={name}
  value={multiple ? (value ?? innerValue).join(",") : value ?? innerValue}
  readOnly
  aria-hidden="true"
/>
const ValueInput = styled.input`
  display: none;
`;
```

Look the Select's implementation for more information.